pyproject.toml

```
@@ -66,7 +66,8 @@ keywords = [
   "gpu",
   "dpcpp"
 ]
-license = {text = "Apache 2.0"}
+license = "Apache-2.0"
```

The license used by `dpnp` is `BSD-2-Clause` which is different than `dpctl` that uses `Apache-2.0`.
```suggestion
license = "BSD-2-Clause"
```